Assets/Live2D/Cubism/Rendering/Masking/CubismMaskRenderer.cs

```diff
@@ -123,6 +123,9 @@ internal void AddToCommandBuffer(CommandBuffer buffer)
             var mainTexture = MainRenderer.MainTexture;
             var mesh = MainRenderer.Mesh;
 
+            //  Exception handling in iOS "ArgumentNullException: Value cannot be null. Parameter name: value"
+            if (mainTexture == null) return;
```

Comment:
```suggestion
            if (mainTexture == null)
            {
                Debug.LogWarning("`mainTexture` is not exist. The rendering will be skipped.");
                return;
            }
```

We have suggested one modification. Once that change is complete, we will be able to incorporate it.